terraform/install.sh

```diff
@@ -60,7 +60,7 @@ getBillingAccount() {
 
 installTerraform() {
   sudo apt-get install unzip
-  wget -q https://releases.hashicorp.com/terraform/0.11.11/terraform_0.11.11_linux_amd64.zip -O ./terraform.zip
+  wget -q https://releases.hashicorp.com/terraform/0.12.29/terraform_0.12.29_linux_amd64.zip -O ./terraform.zip
```

Comment:
@eaberman, FYI - we might want to move installing terraform into your custom cloud shell image.